swagger.yaml

```
@@ -130,6 +130,12 @@ paths:
           type: boolean
           allowEmptyValue: true
           default: false
+        - in: query
+          name: filter_by_owner
+          description: Whether to return only clusters that are owned by the user.
+          type: boolean
```

Maybe it would be better to make the filter generic so that a user can query someone else's resources (in the same org of course)?
```suggestion
          name: owner
          description: Whether to return only clusters that are owned by the user.
          type: string
```

---

good idea, added.

---

Also from the UI point of view it will have to fetch the current user ID which I am not sure it has

---

I've added also organization to the filter, so it's enough as it affects only GET requests.

---

We will not be able to query other organization anyhow so no need to have organization as well.

---

Yeah the org in db request is just for filtering the list.